code/modules/barber/barber_shop.dm

```diff
@@ -54,6 +54,25 @@
 		hair_list[third_id] = third_color
 		src.setup_wig(hair_list)
 
+///Randomized wig for the cargo crate
+/obj/item/clothing/head/wig/spawnable/random
+
+	New()
+		var/list/possible_hairstyles = null
+		if (prob(50))
+			possible_hairstyles = filtered_concrete_typesof(/datum/customization_style, /proc/ismasc)
+		else
+			possible_hairstyles = filtered_concrete_typesof(/datum/customization_style, /proc/isfem)
+		var/datum/customization_style/hair_type = null
+		var/picked_color = rgb(rand(0,255),rand(0,255),rand(0,255))
+		hair_type = pick(possible_hairstyles)
+		first_id = initial(hair_type.id)
+		first_color = picked_color
+		if (prob(33))
+			second_id = initial(hair_type.id)
+			second_color = picked_color
```

Comment:
wouldn't you want a *different* second style here

Reply:
I... how did I miss that :sheltersweat: